twemproxy/tests/compose/docker-compose.yaml

```
@@ -11,31 +10,30 @@ services:
     ports:
       - "6102:6379"
 
-
   etcd0:
     image: quay.io/coreos/etcd:v2.2.0
     volumes:
       - /usr/share/ca-certificates/:/etc/ssl/certs
     ports:
-       - 4001:4001
-       - 2380:2380
-       - 2379:2379
+      - 4001:4001
+      - 2380:2380
+      - 2379:2379
     command: |
-            -name etcd0
-            -advertise-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001,http://etcd0:2379,http://etcd0:4001
-            -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001
-            -initial-advertise-peer-urls http://0.0.0.0:2380
-            -listen-peer-urls http://0.0.0.0:2380
-            -initial-cluster-token etcd-cluster-1
-            -initial-cluster etcd0=http://0.0.0.0:2380
-            -initial-cluster-state new
+      -name etcd0
+      -advertise-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001,http://etcd0:2379,http://etcd0:4001
+      -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001
+      -initial-advertise-peer-urls http://0.0.0.0:2380
+      -listen-peer-urls http://0.0.0.0:2380
+      -initial-cluster-token etcd-cluster-1
+      -initial-cluster etcd0=http://0.0.0.0:2380
+      -initial-cluster-state new
     depends_on:
       - redis1
       - redis2
 
   twemproxy:
     # Version 0.3.0
-    image: jgoodall/twemproxy
+    build: .
```

```suggestion
    build:
      context: .
      args:
        TWEMPROXY_VERSION: ${TWEMPROXY_VERSION}
```

Then set in `tox.ini`